api_testrail.py

```diff
@@ -20,7 +20,7 @@
     ReportTestCaseCoverage,
     ReportTestRailMilestones,
     ReportTestRailUsers,
-    ReportTestRailTestPlans, ReportTestRailTestRuns, ReportTestResultsL10N, ReportTestResultsBeta
+    ReportTestRailTestPlans, ReportTestRailTestRuns, ReportTestResults
```

Comment:
Nit: can you rename following the Report<API><Purpose>? Like: `ReportTestrailTestResults`?